src/App.tsx

```diff
@@ -41,16 +41,14 @@ export const App = () => {
   };
 
   const onChangeHandler = (
-    event: React.ChangeEvent<HTMLInputElement>
-    | React.ChangeEvent<HTMLSelectElement>,
-    type: string,
+    event: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>,
   ) => {
-    if (type === 'title') {
+    if (event.target.tagName === 'INPUT') {
```

Comment:
More common practice to use event.target.name for it. Because in real world can be more large forms with a lot of inputs.
For have here name property you should set name attribute in input.